docs/security/content_trust.md

```diff
@@ -0,0 +1,156 @@
+[menu.main]
+parent= "smn_administrate"
++++
+<![end-metadata]-->
+
+## Enabling content trust in Docker
+
+Docker 1.8 enables content trust. Content trust means that all operations with a remote registry enforce  client-side signing and verification of content tags. Currently content trust is opt-in. We are planning to change this behavior to opt-out in a future release. For now, in order to have trusted operations with Docker you will need to either `export DOCKER_CONTENT_TRUST=1` , or use the `--disable-content-trust=false` flag.
+
+When enabled, content trust will ensure that all operations using a remote registry enforce the signing and verification of tags. In particular `push`, `pull`, `build`, `create` and `run` will only operate on tags that either have content signatures or explicit content hashes.
+
+#### Pulling trusted content
+Let's try to pull a remote repository on the Docker hub that does not have content-trust data:
+
+```
+➜ export DOCKER_CONTENT_TRUST=1
+➜ docker pull diogomonica/alpine
+Using default tag: latest
+no trust data available
+```
+
+As we can see, since we have docker content trust enabled, we are not allowed to pull from a repository that does not have signed content.
+
+#### Pushing trusted content
+
+To create signed content for a specific tag you simply have to push to a repository, providing a content tag. However, if this is the first time you are pushing to a remote repository with content trust enabled, two things will happen:
+
+ - A new root key will be generated and you will be asked to provide a phassphrase for this key
+ - A new tagging pair will be generated, and you will also be asked to provide a passphrase for these keys
+
+```
+# docker push diogomonica/alpine:latest
+...
+You are about to create a new root signing key passphrase.
+You can find the key in your config directory.
+Enter passphrase for new root key with id 3a97f:
+Repeat passphrase for new root key with id 3a97f:
+Enter passphrase for new targets key with id docker.io/diogomonica/alpine/03cf0:
+Repeat passphrase for new targets key with id docker.io/diogomonica/alpine/03cf0:
+Finished initializing "docker.io/diogomonica/alpine"
+```
+
+It is **very important** to note that loss of the root key will require contacting Docker's support team to reset the repository state and will also require **manual intervention** from every docker user that has used the repository before (removing the old root certificate). You should backup this key somewhere safe. We will show you how to backup your keys later on in this document.
+
+Additionally, the passphrase you chose for both the root key and your content key-pair should be randomly generated and stored in a **password manager**.  You can find more details on how these keys are used in the content trust key structure section.
+
+Now that we have pushed the tag `latest`, we can confirm that our trusted pull now works as expected:
+
+```
+➜  docker pull diogomonica/alpine
```

Comment:
:+1: looking at that image on Docker Hub; https://registry.hub.docker.com/u/diogomonica/alpine/

- It's not automated or official 
- Has no description
- There's no Dockerfile or a link to a Dockerfile

In most cases we *discourage* users to run such images, because there's no way to verify what the image does.

If we cannot have an official image for this, perhaps an image under the "docker" repo https://registry.hub.docker.com/repos/docker/, because not all users may know @diogomonica